server/api-server/src/users/users.controller.ts

```diff
@@ -20,6 +24,18 @@ export class UsersController {
   create(@Body() createUserDto: CreateUserDto) {
     return this.usersService.create(createUserDto);
   }
+  @UseGuards(LoggedInGuard)
+  @Get('me')
+  async getUserBySessionId(@Session() session: Record<string, any>) {
+    // console.log(session.userId);
+    if (!session.id) {
+      throw new HttpException('Unauthorized', 401);
```

Comment:
`HttpStatus.UNAUTHORIZED`로 바꾸는게 좋을 것 같습니다.
```suggestion
      throw new HttpException('Unauthorized, HttpStatus.UNAUTHORIZED);
```